src/Domain/Entity/Network.php

```diff
@@ -73,6 +78,18 @@ public function __construct(
             ));
         }
 
+        // Validate array of Services
+        if (is_null($services)) {
```

Comment:
I don't think that's going to work :)

Reply:
no, it broke everything. but i unbroke it, so it's all good now